src/pages/Contributing/Designers/definitionOfDoneReview/index.md

```diff
@@ -0,0 +1,42 @@
+---
+title: 'Definition of Done for Abstract Request Review'
+order:  1
+---
+
+> From a designer point of view, Mozaic is based on Sketch and Abstract. Abstract is a versioning software for design files and allows teams to collaborate with a modern workflow. Like a developer does in GitHub, you need to ask for a review before merging your branches. We wrote some rules to follow before clicking **_Request Review_** on Abstract.
+
+## 1. The symbols have been created following the [definition of Done for symbols](--link--) page.
+We wrote some definition of done rules when you create some symbols in Sketch. Please read these instructions before submitting a review on Abstract. It will make the process way easier for us and we thank you for that 🤘.
+
+## 2. Don't forget to click the `Request Review` button
+![request-review](request-dod.png)
+
+## 3. Don't forget to check if Abstract is `Up to date`
+![up-to-date](up-to-date.png)
+
+## 4. Commits are meaningful
+Commits are a testimony of your work on your files. Use them to make a summary of what's been done during the creation process.
+
+## 5. Branch summary
+Here is a branch summary to enter in the **Summary** section in Abstract when submitting a `Request Review`.
+
+```## PR Checklist
```

Comment:
PR is a github term, it's not better to use an Abstract term ?

Reply:
Yes you should use the full name, and maybe explain what is a PR a little More for those who are not familiar with this.